cms/summary_probe

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+'''
+Probe to check used space.
+'''
+
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == '__main__':
+    try:
+        session = get_session()
+        for rse in list_rses():
+            sources = get_rse_usage(rse['id'], per_account=True)
+            rse_type = session.query(models.RSE.rse_type).filter(models.RSE.id == rse['id']).scalar()
+            rse_type = str(rse_type).split(".", 1)[1]
+            if rse_type == 'TAPE' or 'Test' in rse['rse'] or 'Temp' in rse['rse']:
```

Comment:
> Hi @FernandoGarzon , In your rucio_summary_relative.pdf, values are percentages I guess. Are you sending data in this script as percentages. We may need additional field "rucio managed quota" for each rse, to be able to produce same graph.

I think we won't be able to reproduce that graph. It wasn't that idea in the first place. The idea is actually to create a time distribution of used space by accounts, per rse: https://monit-grafana.cern.ch/d/Qqtk3l-Zz/rucio-probes?viewPanel=76&from=1625590524047&orgId=11&refresh=5m&to=1625676924047&var-dst_rse=All&var-instance=prod&var-vo=cms&var-Country=AT&var-rse_type=&var-source=rucio&var-account=agbet&var-rse=T2_BR_SPRACE.

Also: https://monit-grafana.cern.ch/d/Qqtk3l-Zz/rucio-probes?viewPanel=77&from=1625590524047&orgId=11&refresh=5m&to=1625676924047&var-dst_rse=All&var-instance=prod&var-vo=cms&var-Country=AT&var-rse_type=&var-source=rucio&var-account=agbet&var-rse=T2_BR_SPRACE.

What do you think of those graphs? You can select the RSE you want.

Finally, here I'm sending actual data values, not percentages. Since we can only see one RSE at the time, I don't see the purpose of using percentages.